management-service/src/main/java/pl/pwr/zpi/cluster/service/ClusterService.java

```diff
@@ -27,9 +29,42 @@ public class ClusterService {
     private final ReceiverService receiverService;
     private final MetadataService metadataService;
 
+    public List<ClusterMetadataDTO> getAllClusters() {
+        List<ClusterMetadataDTO> clusters = metadataService.getAllClusters();
+        clusters.forEach(this::setClusterConfigurationForMetadata);
+        return clusters;
+    }
+
+    public void setClusterConfigurationForMetadata(ClusterMetadataDTO metadata) {
+        clusterRepository.findById(metadata.getClusterId()).ifPresentOrElse(configuration -> {
+            metadata.setAccuracy(configuration.getAccuracy());
+            metadata.setUpdatedAtMillis(configuration.getUpdatedAtMillis());
+            metadata.setSlackReceivers(configuration.getSlackReceivers());
+            metadata.setDiscordReceivers(configuration.getDiscordReceivers());
+            metadata.setEmailReceivers(configuration.getEmailReceivers());
+        }, () -> setDefaultClusterConfigurationForMetadata(metadata));
+    }
+
+    private void setDefaultClusterConfigurationForMetadata(ClusterMetadataDTO metadata) {
+        metadata.setAccuracy(Accuracy.LOW);
+        metadata.setUpdatedAtMillis(0L);
```

Comment:
samw questoin about default value - shouldn't be now() and accuracy HIGH?

Reply:
UpdatedAt as explained before, accuracy is changed to HIGH :) Thanks!